### ISSUE TITLE
the problem of frequency_offset

### ISSUE THREAD

**Issue by Dream190802** [proto17/dji_droneid#40]
Hi, in the file of process_file.m , there is a frequency_ offset variable, while also inputting the extract_bursts_from_file function, and you also mentioned in the comment "How far off from DC the signal is in the recording", I would like to know the frequency_offset  is the sampling center frequency of SDR? For example, I used USRP for sampling at 2.4445GHz with a bandwidth of 10MHz and frequency_offset is 2.4445GHz. 
Thank you.

**Comment by proto17** [comment]
It's where in the baseband collect that the signal resides.  So, if the signal you wanted was at 2.4445GHz and that's what you set your SDR to, then the `frequency_offset` variable does not apply to you (leave it at 0).  However, if your SDR has a really bad LO spike and you had to tune up or down before recording then you will need to use `frequency_offset`.  Example: You have a sampling rate of 30.72e6, and you tuned the SDR to 2.4445 GHz + 15 MHz (2.4595 GHz) then you would use a `frequency_offset` value of `15e6` (or maybe `-15e6`, not totally clear to me right now).  

Basically the `frequency_offset` value is there to center the signal in baseband before being processed should there be an offset after the recording.  It is *not* for dealing with very small frequency offsets that just exist in any RF communications as that is handled by the CFO estimation logic.

**Comment by Dream190802** [comment]
Thank you very much for your reply. I roughly understand.
